src/app.ts

```diff
@@ -43,7 +46,12 @@ export const handler = async (event: CloudFrontRequestEvent): Promise<CloudFront
       domain: getHost(request),
       logger,
     })
-  } else if (pathname === (await getResultUri(customerVariables))) {
+  } else if (resultUriRegex.test(pathname)) {
```

Comment:
Can we somehow reduce to one regex call? Currently we call `.test()` and `.match()` it would be better if we only called `.match()`.